### ISSUE TITLE
"Apply screen orientation?" dialog only accepts "Yes" as an answer

### ISSUE THREAD

**Issue by hjek** [foobnix/LibreraReader#1343]
Why ask about applying screen orientation if the only answer is **Yes**? Tapping outside the box does nothing, so there's no implicit **No** answer anywhere; but anyways I'd argue there doesn't need to be a **No** button because it is fine for Librera to just follow the screen orientation settings in the app *without also asking every time the system screen is rotated.*

The dialog should just not be there.

Thank you so much for making the best reader app ever.

**Comment by AmitTzah** [comment]
Yes, agreed. This should just be removed. Very annoying UI decision that doesn't seem to serve any purpose.
